exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/SchemaUtils.kt

```diff
@@ -288,4 +288,51 @@ object SchemaUtils {
             currentDialect.resetCaches()
         }
     }
+
+    /**
+     * Sets the current default schema to [schema]
```

Comment:
Please improve the comment to state what schema will be changed only for a current connection.